CMakeLists.txt

```diff
@@ -6,6 +6,14 @@
 cmake_minimum_required (VERSION 3.13)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules")
 
+# Enable Rust based APIs in C++ SDK.
+set(ENABLE_RUST_IN_BUILD ON CACHE BOOL "Enable Rust based APIs in package")
+set(DISABLE_AMQP OFF CACHE BOOL "Disable AMQP based functionality")
+set(DISABLE_AZURE_CORE_OPENTELEMETRY OFF CACHE BOOL "Disable OpenTelemetry based functionality")
+
+message(STATUS "Disabling Rust temporarily")
+set(ENABLE_RUST_IN_BUILD OFF)
```

Comment:
Sticking with ENABLE_XXXX or DISABLE_XXXX makes it easier to think if things are on off.

Reply:
Ok.